tools/cmake/mbed_greentea.cmake

```diff
@@ -70,6 +82,28 @@ macro(mbed_greentea_add_test)
 
     mbed_set_post_build(${MBED_GREENTEA_TEST_NAME})
 
+    if(NOT ${MBED_OUTPUT_EXT} STREQUAL "")
+        set(MBED_GREENTEA_TEST_IMAGE_NAME "${MBED_GREENTEA_TEST_NAME}.${MBED_OUTPUT_EXT}" CACHE INTERNAL "")
+    else()
+        set(MBED_GREENTEA_TEST_IMAGE_NAME "${MBED_GREENTEA_TEST_NAME}.bin" CACHE INTERNAL "")
```

Comment:
Why are they cache variables? I don't think there's any chance for the user to override the test image name, given that we'll eventually build all tests from the top of mbed-os directly and there will be more than one test names. Or is CACHE for another purpose here?

Reply:
I've removed CACHE as it doesn't really make sense for the user to set this variable.